lib/model/AjvValidator.js

```diff
@@ -141,13 +141,13 @@ function parseValidationError(errors, modelClass) {
       }
     }
 
-    errorHash[key] = [
-      {
-        message: error.message,
-        keyword: error.keyword,
-        params: error.params
-      }
-    ];
+    // More than one error can occur for the same key in Ajv, merge them in the array:
+    const array = errorHash[key] || (errorHash[key] = []);
+    array.push({
+      message: error.message,
+      keyword: error.keyword,
+      params: error.params
+    });
```

Comment:
Isn't this a breaking change because:

the old version always replaced the error with the next one, causing the error to be an array with the last error inside. Now you push errors to the array an therefore the last error is the last item in the array.

I think this could be made a non-breaking change by adding errors in reverse order causing the last error to be the first just like before.

I'd assume people out there are using code like `err.data.myProp[0].message`.

Reply:
OK I agree, although I think it's a bit strange to get the errors in reverse order as they occur in the schema...